.github/workflows/rust.yml

```diff
@@ -30,6 +30,10 @@ jobs:
         with:
           command: check
           args: --all-features --all-targets
+      - uses: actions-rs/cargo@v1
```

Comment:
We should probably avoid `actions-rs` actions, since they're no longer being maintained. We can just use `cargo test --all-features --all-targets` instead.

(If we're doing this, I would ditch the `cargo check` task above, since it's implicit in the `cargo test` invocation.)